src/controller/CHIPDeviceController.h

```diff
@@ -278,19 +265,22 @@ class DLL_EXPORT ChipDeviceController : public SecureSessionMgrCallback, public
     NodeId mLocalDeviceId;
```

Comment:
Note for the future: this starts having 10 data members and growing. We may want to start splitting (I believe Yufeng also is looking at splitting introducing exchange management ... this may go a bit beyond)